lib/EditorJs.tsx

```diff
@@ -68,19 +69,19 @@ class EditorJsContainer extends React.PureComponent<Props> {
       // default tools
       paragraph: {
         class: Paragraph,
-        inlineToolbar: true
+        inlineToolbar: true,
       },
-      ...tools
+      ...tools,
     }
 
     this.instance = new EditorJS({
       tools: extendTools,
-      holder: 'editor-js',
+      holder: this.holderID,
 
       ...(onChange && {
-        onChange: this.handleChange
+        onChange: this.handleChange,
```

Comment:
Please keep the project coding style. (this project using [standard js](https://standardjs.com/)) if you are using vscode, i recommend to install eslint extension.